code/game/objects/structures/displaycase.dm

```diff
@@ -115,6 +117,8 @@
 		. += "[initial(icon_state)]_closed"
 
 /obj/structure/displaycase/attackby(obj/item/W, mob/user, params)
+	if(security_level_locked && GLOB.security_level != SEC_LEVEL_GREEN)
+		req_access = list(security_level_access)
```

Comment:
Needs to be reset when alert is lowered again.

Reply:
Also is there a signal you could register for alert level change, to update it on change?

Reply:
There is no signal I could find, no. 

Updated the case to reset when alert level is lowered. 
Also updated all cases to no longer require ID access to close, to prevent some unusual situations where cases can be permanently locked open, especially if someone decides to implement additional display cases with variable security locks at some point.